src/value.jl

```diff
@@ -94,6 +94,8 @@ function Base.show(io::IO, x::CategoricalValue)
     end
 end
 
+Base.show(io::IO, ::MIME"text/plain", x::CategoricalValue) = Base.show(io, x)
```

Comment:
```suggestion
Base.show(io::IO, ::MIME"text/plain", x::CategoricalValue) = Base.show(io, get(x))
```

Reply:
did you want to unwrap the categorical value?

Reply:
No I want to call the fallback